clients/client/src/main/java/org/projectnessie/client/api/NessieApiV1.java

```diff
@@ -104,6 +105,12 @@ public interface NessieApiV1 extends NessieApi {
   /** Retrieve a diff between two references. */
   GetDiffBuilder getDiff();
 
+  /**
+   * Retrieve all the Unreachable reference heads (commit hash) created due to drop reference or
+   * assign reference operation.
+   */
+  List<String> getUnreachableReferenceHeads() throws NessieNotFoundException;
```

Comment:
All other use cases implement a proper builder pattern - why not here? Lazy?

Reply:
As there are no arguments to set for the builder, I felt like it will look odd to just have get() method.

Also in `getDefaultBranch()`, there was no builder pattern.